loopout.go

```diff
@@ -719,11 +719,23 @@ func (s *loopOutSwap) payInvoiceAsync(ctx context.Context,
 	paymentSuccess := err == nil &&
 		paymentStatus.State == lnrpc.Payment_SUCCEEDED
 
-	if err := s.swapKit.server.ReportRoutingResult(
-		ctx, s.swapInfo().SwapHash, s.swapInvoicePaymentAddr, pluginType,
-		paymentSuccess, int32(attempts), dt.Milliseconds(),
-	); err != nil {
-		s.log.Warnf("Failed to report routing result: %v", err)
+	// RoutingPluginNone is requested for prepays which we don't want to
```

Comment:
It's possible that we also have `RoutingPluginNone` for swap payments if we've got multiple payments going off at the same time, right?

Reply:
Using bool flag now.